### ISSUE TITLE
[SF][ColorPalettePopover][A11y] "More colors..." dialog has no meaningful a11y labels

### ISSUE THREAD

**Issue by Neeeko** [SAP/ui5-webcomponents#5023]
### **Bug Description**
When opening the "More colors" dialog from the ColorPalettePopover using a screen reader, you'll see that no meaningful announcements are made

![Screenshot 2022-04-05 at 16 20 26](https://user-images.githubusercontent.com/7316793/161776564-994b9edb-7fdb-4f00-bb79-71abcd5690b5.png)

### **Expected Behavior**
Screen reader should announce meaningful messages on toggles or different input fields so that screen readers user can understand what's happening.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/ColorPalettePopover/
2. Start a screen reader such as NVDA and open the Dialog
3. Navigate to "More colors" Dialog
4. Navigate among this dialog and see the lack of meaningful announcements

### **Context**
- UI5 Web Components version: latest
- OS/Platform: Windows 10
- Browser: Google Chrome (with NVDA)
- Affected component: ColorPalettePopover

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors

**Comment by nnaydenow** [comment]
Hello colleagues,

Inputs / sliders in ui5-color-pallete don't provided meaningful information when they are read by the screen reader.

Regards,
Nayden

**Comment by rchrdchn** [comment]
Hey guys - following up here. Any updates on when a potential fix will be available?

**Comment by unazko** [comment]
Hello @rchrdchn,

We should comply with the accessibility specification and provide meaningful labels the input fields and the two sliders.
Currently we need to enable providing an accessible name for the sliders before we could be able to comply with the specification in the ui5-colorpicker.

Expect updates soon.

Best Regards,
Boyan Rakilovski

**Comment by unazko** [comment]
Hello @rchrdchn,

The related fix is now merged and as a result there will be a meaningful announcements by the screen readers for the input fields and the two sliders.

Best Regards,
Boyan Rakilovski
